tsconfig.json

```diff
@@ -1,9 +1,8 @@
 {
-  // TODO: Use tsconfig/base after all tools support tsconfig.extends array
-  "extends": "./tsconfig.base.json",
+  "extends": ["@tsconfig/strictest/tsconfig", "@tsconfig/node20/tsconfig"],
   "compilerOptions": {
-    "lib": ["ESNext", "DOM"],
+    "lib": ["dom", "dom.iterable"],
```

Comment:
* Passed transpile without this change, but updated with referring https://github.com/tsconfig/bases/blob/014e8927b7155640f65a7492fb4bb09f5c49f531/bases/create-react-app.json#L7